node.wake

```diff
@@ -45,7 +45,7 @@ global def nodeInstaller packageDir Unit =
 #################################################################
 # Create a command for running a node module
 #################################################################
-global def nodeCommand module args  =
+global def nodeCommand _module args  =
     "node", "-e", "module", "--", args
```

Comment:
Wait, is this a bug? I'm pretty sure this function doesn't make any sense if you don't specify the module itself? My guess is that the string `"module"` should just be the module name.

That said, if this is clearly broken and no one is using it, I'd be happy to totally delete this function.

Reply:
I just did a bit of testing with NodeJS, and this is even more broken than the `module` variable not being passed in. [`node -e`](https://nodejs.org/api/cli.html#cli_e_eval_script) allows you to evaluate the next argument as a string that encodes a NodeJS script, similar to `bash -c`, `python -c`, and `wake -x`.

I'm not actually sure what this function was meant to do, and I think it's probably best to just delete it, since it's broken for more than one reason.